cloudweatherreport/cloud_weather_report.py

```diff
@@ -195,7 +195,7 @@ def main(args, test_plan):
             continue
         last_successful_status = status
         benchmark_results = []
-        if test_plan.get('benchmark'):
+        if status is not None and test_plan.get('benchmark'):
```

Comment:
Would it be better to check for an error in "test_results" here? If both test_results and status are None, we'll never get around to executing this line of code in this iteration of the loop (see line 194 above). The case that we care about is when status is None, but test_results contains something like an error message. 

I think that the purpose of this check would be clearer if we explicitly checked the tests_results, rather than skipping based on status.